web-components/src/components/button/button.component.ts

```diff
@@ -123,7 +123,7 @@ export default class Button extends BaseElement {
   }
 
   private get buttonSpacing(): ButtonSpacing {
-    if (this.type === 'icon-only') {
+    if (this.type === 'icon-only' && this.spacing === 'text') {
```

Comment:
Not sure if I understand this change. What's the use case for type 'icon-only' but spacing 'text'?

Reply:
Can't fully remember, it's a change we did in the project (last October 😉). As far as I understand it, the `type` "icon-only" should only affect `spacing` if that has not been set explicitly, which would mean something else than "text" as that's the default.